scout/adapter/mongo/hgnc.py

```diff
@@ -347,15 +347,13 @@ def genes_by_alias(self, build=None, genes=None):
 
         for gene in genes:
             hgnc_id = gene["hgnc_id"]
-            hgnc_symbol = gene["hgnc_symbol"]
 
             for alias in gene["aliases"]:
                 if alias not in alias_genes:
                     alias_genes[alias] = {"true": None, "ids": set()}
 
                 alias_genes[alias]["ids"].add(hgnc_id)
-                if alias == hgnc_symbol:
-                    alias_genes[alias]["true"] = hgnc_id
+                alias_genes[alias]["true"] = hgnc_id
```

Comment:
This code is not safe since it will set the value to the latest HGNC encountered in the loop. I'll close this so we can use your solution, which is the safest!